features/claim/claimFormDisconnected.tsx

```diff
@@ -18,7 +19,7 @@ export const ClaimFormDisconnected = () => {
           placeholder="0"
         />
       </InputGroupStyled>
-      <WalletConnect fullwidth />
+      <WalletConnectButton fullwidth />
```

Comment:
Sad to say, but this should be renamed. Sorry for not seeing this earlier.
We should avoid confusion with WalletConnect.

Reply:
Do you have suggests how rename?

Reply:
done.
https://github.com/lidofinance/warehouse/pull/159/commits/b1c5c6e436598a76519827993f5cddd4abe80752

Reply:
WalletConnectButton  ---> ConnectWalletButton